### PR TITLE
*UW Student Editor* Updated data-systems.md

### PR DESCRIPTION
**Pull request by HennaSheikh**
Edited "Data Paths" section (up until next major section [Data Flow]) for grammar, consistency, and general copy errors.

### CODE REVIEW

**Comment by HennaSheikh** [comment]
I have signed the CLA.

Also, Arthur if you're reading this, I hope it worked this time!

File: app/1.0/docs/devguide/data-system.md [hunk 1]
```diff
@@ -237,11 +237,11 @@ a wildcard path in a data binding.
 #### Array mutation paths
 
 When used as the last segment in a path, `splices` represents any array *mutations* to the
-identified array (additions or deletions). For example, if `users` is an array, the path
-`users.splices` identifies any additions to or deletions from the array.
+identified array (i.e. additions or deletions). For example, if `users` is an array, the path
```
Comment (arthurevans) [inline app/1.0/docs/devguide/data-system.md]: We don't use i.e. or e.g.—"that is" or "for example" are fine.

File: app/1.0/docs/devguide/data-system.md [hunk 2]
```diff
@@ -268,7 +268,7 @@ because it requires extra work to retrieve the index. Polymer provides methods t
 original array item given the item's key.
 
 **In most cases, you don't need to deal with array keys directly**. Polymer provides helper
-elements, such as the [template repeater element](/1.0/docs/devguide/templates#dom-repeat) that
+elements, such as the [template repeater element](/1.0/docs/devguide/templates#dom-repeat), that
```
Comment (arthurevans) [inline app/1.0/docs/devguide/data-system.md]: Here, **that** introduces a [restrictive clause](http://www.quickanddirtytips.com/education/grammar/which-versus-that-0) (not all helper elements are involved with arrays). So no comma.

File: app/1.0/docs/devguide/data-system.md [hunk 3]
```diff
@@ -250,15 +250,15 @@ registered by a wildcard path (for example, you won't see changes to subproperti
 Polymer supports two ways of identifying an array item in a path: by index or by an opaque,
 immutable key.
 
-*   An index, for example, `"myArray.1"` indicates the array item at position 1.
+*   An index, for example, `"myArray.1"`, indicates the array item at position 1.
```
Comment (arthurevans) [inline app/1.0/docs/devguide/data-system.md]: You're right that the example here is an appositive, and should be set off as such... But the extra comma makes it a little hard to read. Might be better here to set off the appositives with an em dash, like:

An index—for example, `"myArray.1"`—indicates the... 

That way, the comma after "for example" is unambiguous. 

Also, while you're in there, "indicates" should really be "identifies". I don't know what I was thinking, there.

File: app/1.0/docs/devguide/data-system.md [hunk 4]
```diff
@@ -250,15 +250,15 @@ registered by a wildcard path (for example, you won't see changes to subproperti
...
 
-*   An opaque key, prefixed with the pound sign (`#`), for example, `"myArray.#1"` indicates the
+*   An opaque key, prefixed with the pound sign (`#`), for example, `"myArray.#1"`, indicates the
```
Comment (arthurevans) [inline app/1.0/docs/devguide/data-system.md]: Same comment as above.

**Comment by arthurevans** [comment]
Worked great, thanks! Reviewed and left comments.

**Comment by HennaSheikh** [comment]
Update: I made the changes to my PR, as per your comments.

**Comment by arthurevans** [comment]
Thanks @HennaSheikh ... You're not showing up as having signed the CLA. Did you sign it using the email address associated with your github account?

**Comment by HennaSheikh** [comment]
My apologies for the delay, Arthur.  I've just updated the CLA with my GitHub-associated email address.  I assume this may take some time to process?

**Comment by arthurevans** [comment]
Hi @HennaSheikh ... I'm afraid I've pushed a bit set of updates to the branch, so there are now merge conflicts. If you want to continue with this PR, you'd need to rebase against the current data-system branch.

**Pull request closed without merging**